src/main/java/jalau/at18/architects/BluePoints.java

```diff
@@ -0,0 +1,20 @@
+package jalau.at18.architects;
+
+public class BluePoints {
+    private int points = 0;
+    private int cats = 0;
+
+    public void setPoints(BlueCard blueCard) {
```

Comment:
I'd suggest change the name to `addPoints` instead of `setPoints`, this because `points` are being increased with points instead of being replaced with a new value. Typically, "set" is used when a value is replaced with a new one.